contracts/registries/IPAssetRegistry.sol

```diff
@@ -34,12 +36,16 @@ contract IPAssetRegistry is
     using ERC165Checker for address;
     using Strings for *;
     using IPAccountStorageOps for IIPAccount;
+    using SafeERC20 for IERC20;
 
     /// @dev Storage structure for the IPAssetRegistry
     /// @notice Tracks the total number of IP assets in existence.
     /// @custom:storage-location erc7201:story-protocol.IPAssetRegistry
     struct IPAssetRegistryStorage {
         uint256 totalSupply;
+        address treasury;
```

Comment:
Do we need natspec for the storage struct with description of each variable?

Reply:
Added comments.